dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe

```diff
@@ -18,8 +18,9 @@ elif [ "$targetArchitecture" = x86_64 ]; then
 	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
 fi
 
-ARCHITECTURES="all !x86_gcc2"
+ARCHITECTURES="!all x86_64"
 SECONDARY_ARCHITECTURES="x86"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
Looks correct now @korli ?

Reply:
Looks good!

Reply:
The lint fails: please add an empty SOURCE_URI for all arches.